proto/frequenz/api/microgrid/common.proto

```diff
@@ -11,6 +11,7 @@ syntax = "proto3";
 package frequenz.api.microgrid.common;
 
 // A set of lower and upper bounds for any metric.
+// Bounds units are always the same as the related metric.
```

Comment:
I'd rephrase this slightly:
`Bounds units` => `Units of the bounds`

Reply:
Done